### PR TITLE
Style everything

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: css/site.styl [hunk 1]
```diff
@@ -50,51 +50,121 @@ img
...
+// ******************************************************
+
+// MAIN CONTENT
+
+#main-content
+  position: fixed
+  top: 0
+  left: 0
+  width: 100%
+  height: 100%
```
Comment (varffs) [inline css/site.styl:116]: I think 100vh is better than % in general

File: js/main.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ Site = {
       _this.onResize();
     });
 
+    _this.thinkOf();
```
Comment (varffs) [inline js/main.js:13]: I think this should have been another feature. It isn't related to style everything

**Review by varffs — COMMENTED** [review]
Works well and looks as it should just a few general comments

File: css/site.styl (hunk 1 above)
Comment (mikeyrayvon) [reply css/site.styl:116]: but what about on mobile? 100vh fails on mobile cuz of the browser bar

File: js/main.js (hunk 2 above)
Comment (mikeyrayvon) [reply js/main.js:13]: i needed it to style the footer, breh

File: css/site.styl (hunk 1 above)
Comment (varffs) [reply css/site.styl:116]: i think it is the other way round? `100%` was the cause of a bug on mc-h

Comment (mikeyrayvon) [reply css/site.styl:116]: hmm. im pretty sure it was `vh` that caused it. @mixcas ?

Comment (varffs) [reply css/site.styl:116]: i fixed it (well the jumping bug) and it was caused by % and vh fixed it

**Pull request merged**